### ISSUE TITLE
What exactly is the validation set?

### ISSUE THREAD

**Issue by vidit09** [timy90022/One-Shot-Object-Detection#9]
@timy90022 thanks for the code repo. I have a question regarding the validation set used during training. In the [code](https://github.com/timy90022/One-Shot-Object-Detection/blob/master/trainval_net.py#L163), it looks like test set is used for validation. Could you please make it clear what exactly is the strategy here. Thanks.
